core/modules/wmgr/python/xrdMgr.py

```diff
@@ -120,6 +122,46 @@ def _restartService(sName):
 def _restartXrootd():
     _restartService('xrootd')
 
+
+def _workerId():
+    """ Fetch and return a unique identifier of the worker. """
+
+    _log.debug('fetching a unique identifier of the worker')
+
+    dbName = 'qservw_worker'
+    tblName = 'Id'
+
+    # check the db exists (listTables() does not fail on non-existing databases)
+    with Config.instance().dbEngine().begin() as dbConn:
+        if not utils.tableExists(dbConn, tblName, dbName):
+            raise ExceptionResponse(404, "TableMissing", "Table %s.%s does not exist" % (dbName, tblName))
+
+        query = "SELECT id FROM `%s`.`%s`" % (dbName, tblName)
+        rows = dbConn.execute(query)
+
+        row = rows.first()
+        if row is None:
+            raise ExceptionResponse(404, "NowWorkerIdFound",
+                                    "Table %s.%s does not have any worker identifier" % (dbName, tblName))
+        return row[0]
+
+
+def _rebuildChunkTable():
+    """ Rebuild and reload a list of databases and chunks w/o restarting a service """
+
+    workerId = _workerId()
+
+    _log.info('rebuilding a list database and chunks known to worker %s', workerId)
+
+    try:
+        cmd = ['qserv-worker-notify','REBUILD_CHUNK_LIST',workerId,'--reload']
+        _runCmd(cmd, False)
+    except subprocess.CalledProcessError as exc:
+        raise ExceptionResponse(409, "CommandFailure",
+                                "Failed to rebuild a list database and chunks known to worker %s" % workerId,
```

Comment:
"list database" -> "list of databases" ?

Reply:
Fixed